### ISSUE TITLE
Typo 'contasts' in Understanding SC 1.4.11 (WCAG 2.1)

### ISSUE THREAD

**Issue by cstrobbe** [w3c/wcag#1996]
The caption below figure 5 contains the following typo:

> a color that **contasts** with ...

Obviously, that should be "contrasts".

**Comment by alastc** [comment]
Thanks, I've corrected that for 2.1 and Main.
